tests/symengine/symengine_number_type_01.output

```diff
@@ -0,0 +1,119 @@
+DEAL:Float:Math operators::a-b: -4.0
+DEAL:Float:Math operators::a*b: 45.0
+DEAL:Float:Math operators::a/b: 0.555555555555556
+DEAL:Float:Math operators::c+=b: 14.0
+DEAL:Float:Math operators::c-=b: 5.0
+DEAL:Float:Math operators::c*=b: 45.0
+DEAL:Float:Math operators::c/=b: 5.0
+DEAL:Float:Math operators with normal numbers::c+=s: 7.0
+DEAL:Float:Math operators with normal numbers::c*=s: 14.0
+DEAL:Float:Math operators with normal numbers::c-=s: 12.0
+DEAL:Float:Math operators with normal numbers::c/=s: 6.0
+DEAL:Float:Math operators with normal numbers::x: x
+DEAL:Float:Math operators with symbols::c+=x: 6.0 + x
+DEAL:Float:Math operators with symbols::c*=x: x*(6.0 + x)
+DEAL:Float:Math operators with symbols::c-=x: -x + x*(6.0 + x)
+DEAL:Float:Math operators with symbols::c/=x: (-x + x*(6.0 + x))/x
+DEAL:Float:Differentiation::dc_dx: x*(-1 + x + 6.0 + x) - x + x*(6.0 + x)
+DEAL:Float:Substitution::dc_dx(x=2): 32.0
+DEAL:Float:Substitution::dc_dx(x=2): 32.0
+DEAL:Float:Substitution::dc_dx(x=3.5): 71.75
+DEAL:Float:Substitution::dc_dx(x=y**2+3): (3 + y**2)*(9.0 + y**2) + (3 + y**2)*(11.0 + 2*y**2) - (3 + y**2)
+DEAL:Float:Substitution::dc_dx(x=y**2+3): (3 + y**2)*(9.0 + y**2) + (3 + y**2)*(11.0 + 2*y**2) - (3 + y**2)
+DEAL:Float:Substitution::dc_dx(x=y**2+3;y=1.5) [non-recursive]: (3 + y**2)*(9.0 + y**2) + (3 + y**2)*(11.0 + 2*y**2) - (3 + y**2)
+DEAL:Float:Substitution::dc_dx(x=y**2+3;y=1.5) [recursive]:     135.1875
+DEAL:Float:Parsing::dp1_dx: 2.0*x**3
+DEAL:Float:Parsing::dp2_dx: 2.0*x**3
+DEAL:Double:Initialize::a: 10.0
+DEAL:Double:Initialize::b: 10.0
+DEAL:Double:Relational operators 1::a == b: 1
+DEAL:Double:Relational operators 1::a != b: 0
+DEAL:Double:Relational operators 1::a < b: 0
+DEAL:Double:Relational operators 1::a > b: 0
+DEAL:Double:Set new values::a: 5.0
+DEAL:Double:Set new values::b: 9.0
+DEAL:Double:Relational operators 2::a == b: 0
+DEAL:Double:Relational operators 2::a != b: 1
+DEAL:Double:Relational operators 2::a < b: 1
+DEAL:Double:Relational operators 2::a > b: 0
+DEAL:Double:Math operators::a+b: 14.0
+DEAL:Double:Math operators::a-b: -4.0
+DEAL:Double:Math operators::a*b: 45.0
+DEAL:Double:Math operators::a/b: 0.555555555555556
+DEAL:Double:Math operators::c+=b: 14.0
+DEAL:Double:Math operators::c-=b: 5.0
+DEAL:Double:Math operators::c*=b: 45.0
+DEAL:Double:Math operators::c/=b: 5.0
+DEAL:Double:Math operators with normal numbers::c+=s: 7.0
+DEAL:Double:Math operators with normal numbers::c*=s: 14.0
+DEAL:Double:Math operators with normal numbers::c-=s: 12.0
+DEAL:Double:Math operators with normal numbers::c/=s: 6.0
+DEAL:Double:Math operators with normal numbers::x: x
+DEAL:Double:Math operators with symbols::c+=x: 6.0 + x
+DEAL:Double:Math operators with symbols::c*=x: x*(6.0 + x)
+DEAL:Double:Math operators with symbols::c-=x: -x + x*(6.0 + x)
+DEAL:Double:Math operators with symbols::c/=x: (-x + x*(6.0 + x))/x
+DEAL:Double:Differentiation::dc_dx: x*(-1 + x + 6.0 + x) - x + x*(6.0 + x)
```

Comment:
the derivative here looks wrong

Reply:
Wow, lucky catch! I still needed to update this tests's blessed output, which would have overwritten this :-)